dump.rdb

```diff
@@ -0,0 +1 @@
+REDIS0006�ܳC�Z��V
```

Comment:
I don't think this is supposed to be here :smile:

Reply:
Ugh, sorry for noise :bowing_man:  I removed.